### PR TITLE
Small tweaks: value for an option, missing quotes, a few typos

### PR DESCRIPTION
**Pull request by mslw**
Hi, and thank you for this nice tutorial! Here's a PR fixing two small things which gave me errors during the walk-through - it seems that at least one of them is due to a change in one of the previous jj releases.

I also used the opportunity to make some typo changes: two word replacements for consistency, and a few typos found with codespell.

### CODE REVIEW

**Comment by steveklabnik** [comment]
Ah, I read #34 before this, let's include this in the plans over here https://github.com/steveklabnik/jujutsu-tutorial/pull/34#issuecomment-2414582897

**Comment by sogaiu** [comment]
Thanks for the "Add missing value '@' for jj new -B" bit.

I should have checked here before messing up my tutorial state (^^;

Perhaps doing things over isn't bad for practice though.

**Review by steveklabnik — APPROVED** [review]
I *really* need to figure out how to integrate spellcheck in VS: Code, haha. Thank you so much.

File: src/sharing-code/updating-prs.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ We can see that reflected on GitHub:
 
 ![a screenshot of github showing that our branch was force pushed](../images/force-push-pr.png)
 
-Only one commit, and it shows that our comment is on an outdated commit.
+Only one commit, and it shows that our comment is on an outdated comment.
```
Comment (rmacklin) [inline src/sharing-code/updating-prs.md:177]: I don't think this change is correct. The original wording makes sense, "comment is on an outdated comment" doesn't make sense to me.

Comment (rmacklin) [reply src/sharing-code/updating-prs.md:177]: I've opened https://github.com/steveklabnik/jujutsu-tutorial/pull/47 to revert this

Comment (rmacklin) [reply src/sharing-code/updating-prs.md:177]: Ultimately changed it to "comment is on an outdated diff" to be clearer/more accurate.

**Pull request merged**